web/app/themes/frocentric-tech/css/elements.css

```diff
@@ -255,6 +255,7 @@ aside.cite {
 
 .footer-widgets {
   padding-bottom: 0;
+  margin-top: 10rem;
```

Comment:
This rule affects every page and breaks the layout on some where the content is expected to push right up against the footer. This would be better relocated to the `#main .inside-article` selector edited below, which already has a `margin-bottom` defined that could simply be increased to the value applied here.